runtime/router.go

```diff
@@ -205,6 +211,11 @@ func NewRouter(gateway *Gateway) *Router {
 }
 
 func (router *Router) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	err := router.logRequest(r)
```

Comment:
I think it would be nice to log res status code, should do this after `ServeHTTP()` ?